app/src/main/java/ch/hikemate/app/model/route/ListOfHikeRoutesViewModel.kt

```diff
@@ -51,6 +51,26 @@ open class ListOfHikeRoutesViewModel(
   private suspend fun getRoutesAsync(onSuccess: () -> Unit = {}, onFailure: () -> Unit = {}) {
     withContext(dispatcher) {
       val area = area_.value ?: return@withContext
+
+      // Check if the area is on the date line
+      if (area.lonEast < area.lonWest) {
```

Comment:
Suggestion: create an extension method on `BoundingBox` as you did with `Bounds` to make this condition easier to read.

```kotlin
fun BoundingBox.isCrossingDateLine(): Boolean {
    return lonEast < lonWest
}
```

Reply:
Nice!